VAMobile/e2e/tests/utils.ts

```diff
@@ -243,6 +242,26 @@ export async function openVaccineRecords() {
   await element(by.text(CommonE2eIdConstants.VACCINE_RECORDS_BUTTON_TEXT)).tap()
 }
 
+export async function showOnboarding() {
```

Comment:
Since this duplicates almost all of loginToDemoMode, you could instead add an argument to loginToDemoMode like:

```
export async function loginToDemoMode(skipCarousel = true) {
```

And then you could add a conditional on the last bit of the function, or return early, depending on that argument's value. The argument would default to true so it wouldn't affect existing calls of the function.